dashboard/views.py

```diff
@@ -41,7 +41,7 @@ def get(self, request):
               title='Timeseries')
         graphJSON = json.dumps(fig, cls=plotly.utils.PlotlyJSONEncoder)
         
-        context = {'all_metrics':all_metrics, "current_metrics":current_metrics, "total_warnings":total_warnings, "graphJSON":graphJSON}
+        context = {'all_metrics':all_metrics, "current_metrics":current_metrics, "total_warnings":total_warnings[:6], "graphJSON":graphJSON}
```

Comment:
rather than limiting the warning here, it should be more effective to limit it in the query. Reason being the query will only fetch that many items and will be efficient in comparison to fetching 'everything' and then limiting it.

```
total_warnings_qs = CAPInstance.objects.values("cap_id", "date", "total_warnings").order_by("date").reverse()[:limit]
```

Reply:
Adjusted Query to limit results